### ISSUE TITLE
Disappearing Object Bug - Sentry/SecCam/Search Light Behaviors

### ISSUE THREAD

**Issue by Blood-Moon-Interactive** [TheGameCreators/GameGuruRepo#4864]
Using the Sentry, Security Camera, or Search Light behaviors on the included test objects work perfectly fine. However if you add any of these behaviors to any random object and run the test level, the object will disappear! In the video I will demonstrate this peculiar bug. I will also send an example of the level and the custom model created by Glitched Pixels via email directly to Lee for testing.

Demonstration Video:
https://youtu.be/igvsEC1KxDY

**Comment by LeeBamberTGC** [comment]
@Blood-Moon-Interactive The issue was in the script, hiding(destroying) the object when it reached a health of 1. As the object was already set to 1, boom :)  I have updated the script for the next EXP build and emailed the change to Necrym.  I have also updated the UI so explodable properly hides when an object is immobile, previously it only read the original setting for isimmobile rather than the dynamic one set by the user.
